### PR TITLE
reducing length of privacy considerations

### LINKED ISSUE #98: Privacy Considerations would benefit from re-write
Privacy Considerations is long and worries about some of the wrong things, like modifying the metadata by an on-path attacker.  This section should have a structure of:

1. traffic analysis is a problem for user privacy (e.g., ISP builds profile of streamed movies)
2. existing TLS, SRTP, and QUIC traffic is vulnerable to traffic analysis, and have some mitigations like padding
3. per-packet metadata makes traffic analysis easier
4. mitigations for (3) are:  encrypting/obfuscating per-packet metdata, mis-marking metadata
5. however, encrypting/obfuscating per-packet metadata is ineffective if threat is from the same network entity with the keys

### PR DESCRIPTION
**Pull request by sridharangirish91**

### CODE REVIEW

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 1]
```diff
@@ -705,54 +721,29 @@ metadata.
...
-allow unauthorized entities to infer sensitive information about the
-data being transmitted while maintaining minimal resource
-consumption. There is a tension between resource consumption (and thus efficiency) of such
-encryption and the user's privacy and this tension depends on the threat
-model ({{Section 7.4 of ?RFC6973}}); the threat of a network provider
-building a subscriber profile of viewed video content
-is different from the threat of an interactive voice or video call. To
-mitigate traffic analysis, the sender might, for example, purposefully mis-mark
-metadata in some packets, add some randomness to avoid recurrent traffic patterns, etc.
+Media flows are vulnerable to traffic analysis without per packet
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
Media flows are vulnerable to traffic analysis even without per-packet
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 2]
```diff
@@ -705,54 +721,29 @@ metadata.
...
+metadata ({{traffic-analysis}}). The security aspects of the
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
metadata (see, e.g., {{traffic-analysis}}). The security aspects of the
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 3]
```diff
@@ -705,54 +721,29 @@ metadata.
...
+media payload / transport are not in the scope of this document and
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
media payload / transport are not in the scope of this document; these
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 4]
```diff
@@ -705,54 +721,29 @@ metadata.
...
+is mentioned here only to provide context for metadata privacy.
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
are mentioned here only to provide context for metadata privacy.
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 5]
```diff
@@ -705,54 +721,29 @@ metadata.
...
+
+Existing TLS, SRTP and QUIC protocols have some mititgations (like padding)
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
Protocols such as TLS, SRTP, and QUIC offer some mitigations (like padding)
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 6]
```diff
@@ -705,54 +721,29 @@ metadata.
...
+but are vulnerable to traffic analysis. ({{traffic-analysis-2}})
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
but are vulnerable to traffic analysis ({{traffic-analysis-2}}).
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 7]
```diff
@@ -705,54 +721,29 @@ metadata.
...
+Metadata is vulnerable to modification in transit by
+on-path attackers, who can corrupt checksums, drop packets, or mislabel
+important packets. Such changes are detectable by the receiver.
+
+Per-packet metadata can aid in traffic analysis. Hence, it is recommended to
+encrypt or obfuscate the metadata information so it is only available to the
+server, client, and authorized network elements. However, encryption/obfuscation
+of per-packet metadata is ineffective if the threat resides in the same network
+entity with keys to decrypt the metadata. The method of encryption or
+obfuscation is out of this document's scope.
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
obfuscation is out of scope.
```

**Review by boucadair — CHANGES_REQUESTED** [review]

**Review by boucadair — APPROVED** [review]

**Comment by John-Kaippallimalil** [comment]
Changes look good.

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 8]
```diff
@@ -694,53 +710,30 @@ metadata.
...
+Media flows are vulnerable to traffic analysis even without per-packet
+metadata (see, e.g., {{traffic-analysis}}). The security aspects of the
+media payload / transport are not in the scope of this document; these
+are mentioned here only to provide context for metadata privacy.
+
+Protocols such as TLS, SRTP, and QUIC offer some mitigations (like padding)
+but are vulnerable to traffic analysis ({{traffic-analysis-2}}).
+Metadata is vulnerable to modification in transit by
+on-path attackers, who can corrupt checksums, drop packets, or mislabel
+important packets. Such changes are detectable by the receiver.
```
Comment (danwing) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md:722]: Modifying metadata in transit seems a security concern, not a privacy concern. Should this be moved to security considerations?  Also, CIDFI (as one per-packet metadata mechanism we could consider) prevents such modifications, so this blanket statement needs more nuance.

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 9]
```diff
@@ -694,53 +710,30 @@ metadata.
...
+server, client, and authorized network elements. However, encryption/obfuscation
+of per-packet metadata is ineffective if the threat resides in the same network
+entity with keys to decrypt the metadata. The method of encryption or
+obfuscation is out of scope.
+
+Analysis to ensure that metadata exposure does not compromise user privacy
+or allow unauthorized entities to infer sensitive information, while
+maintaining minimal resource consumption is crucial. There is a tension
+between resource consumption of such encryption and the user's privacy
+({{Section 7.4 of ?RFC6973}}).
```
Comment (danwing) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md:735]:
```suggestion
({{Section 7.4 of ?RFC6973}}).

To best preserve privacy, implementations might also consider less granular per-packet marking, for example marking all audio and video packets the same and only marking a background data transfer with different metadata.
```

**Review by danwing — DISMISSED** [review]
one addition and one move of paragraph to Security and I'm good.

**Pull request merged**